app/assets/stylesheets/application.scss

```diff
@@ -34,14 +55,189 @@ footer {
+//     animation: animate2 2s linear infinite;
+//     animation-delay: 1s;
+    
+// }
+// @keyframes animate2 {
+//     0% {
+//         transform: translateY(-100%);
+//     }
+//     100% {
+//         transform: translateY(100%);
+//     }
+// }
+
+// .container .card span:nth-child(3) {
+//     position: absolute;
+//     bottom: 0;
+//     left: 0;
+//     width: 100%;
+//     height: 3px;
+//     background: linear-gradient(to left, transparent, red);
+//     animation: animate3 2s linear infinite
+// }
+// @keyframes animate3 {
+//     0% {
+//         transform: translateX(100%);
+//     }
+//     100% {
+//         transform: translateX(-100%);
+//     }
+// }
+
+// .container .card span:nth-child(4) {
+//     position: absolute;
+//     top: 0;
+//     left: 0;
+//     width: 3px;
+//     height: 100%;
+//     background: linear-gradient(to top, transparent, red);
+//     animation: animate4 2s linear infinite;
+//     animation-delay: 1s;
+    
+// }
+// @keyframes animate4 {
+//     0% {
+//         transform: translateY(100%);
+//     }
+//     100% {
+//         transform: translateY(-100%);
+//     }
+// }
+
+// .container .card span {
+//     transition: 0.5;
+// }
+
+// .container .card span:nth-child(1) {
+//     position: absolute;
+//     top: 0;
+//     left: 0;
+//     width: 100%;
+//     height: 3px;
+//     background: linear-gradient(to right, transparent, red);
+//     animation: animate1 2s linear infinite
+// }
+// @keyframes animate1 {
+//     0% {
+//         transform: translateX(-100%);
+//     }
+//     100% {
+//         transform: translateX(100%);
+//     }
+// }
+
+// .container .card span:nth-child(2) {
+//     position: absolute;
+//     top: 0;
+//     right: 0;
+//     width: 3px;
+//     height: 100%;
+//     background: linear-gradient(to bottom, transparent, red);
+//     animation: animate2 2s linear infinite;
+//     animation-delay: 1s;
+    
+// }
+// @keyframes animate2 {
+//     0% {
+//         transform: translateY(-100%);
+//     }
+//     100% {
+//         transform: translateY(100%);
+//     }
+// }
 
+// .container .card span:nth-child(3) {
+//     position: absolute;
+//     bottom: 0;
+//     left: 0;
+//     width: 100%;
+//     height: 3px;
+//     background: linear-gradient(to left, transparent, red);
+//     animation: animate3 2s linear infinite
+// }
+// @keyframes animate3 {
+//     0% {
+//         transform: translateX(100%);
+//     }
+//     100% {
+//         transform: translateX(-100%);
+//     }
+// }
 
+// .container .card span:nth-child(4) {
+//     position: absolute;
+//     top: 0;
+//     left: 0;
+//     width: 3px;
+//     height: 100%;
+//     background: linear-gradient(to top, transparent, red);
+//     animation: animate4 2s linear infinite;
+//     animation-delay: 1s;
+    
+// }
+// @keyframes animate4 {
+//     0% {
+//         transform: translateY(100%);
+//     }
+//     100% {
+//         transform: translateY(-100%);
+//     }
+// }
```

Comment:
This is a lot of commented-out code.  Please either comment it back in, or remove it.